src/App.tsx

```diff
@@ -1,18 +1,134 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Navigate, Route, Routes, useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { getPeople } from './api';
+import { Person } from './types';
+import { SortParams } from './components/types';
 
 export const App = () => {
+  const [loadingPeople, setLoadingPeople] = useState<boolean>(true);
+  const [error, setError] = useState<boolean>(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [visiblePeople, setVisiblePeople] = useState<Person[]>([]);
```

Comment:
That means that you can use `let` or `const` for this information 

Like this `const visiblePeople = handleSortFunction(your attributes)`